reports/report_emailer.py

```diff
@@ -63,22 +43,27 @@ def average_delay_per_hour_graph(conn: psycopg2.extensions.connection, station_c
                 y=[i[1] for i in delays_per_hour_data])
     plt.xlabel("Hour of the Day")
     plt.ylabel("Average Delay Time")
+    plt.title("Average Delay Time for Delayed Trains")
```

Comment:
Would be good to specify a time period here. The reports are weekly, so it'd be good for the function to only use data from the last week.